content/ja/examples/minikube/server.js

```diff
@@ -1,9 +1,9 @@
 var http = require('http');
 
 var handleRequest = function(request, response) {
-  console.log('Received request for URL: ' + request.url);
+  console.log('URLのリクエストを受信しました ' + request.url);
   response.writeHead(200);
-  response.end('Hello World!');
+  response.end('こんにちは世界');
```

Comment:
Especially this change is too much while the other change would be "okay". 

"Hello World" is enough popular and known concept for any programmers in Japan.